test/acceptance/animated.test.js

```diff
@@ -0,0 +1,150 @@
+import React from 'react'
+import Counter from '../../src/'
+import { render, renderAnimatedTenSecondCounter } from './support/helpers'
+
+// Unmocked setTimeout
+const timeout = setTimeout
+
+describe('ticking', function () {
+  it('changes as time passes', function () {
+    const component = renderAnimatedTenSecondCounter()
+    expect(component).toDisplayDigits('10')
+    jasmine.clock().tick(1000)
+    expect(component).toDisplayDigits('09')
+    jasmine.clock().tick(3000)
+    expect(component).toDisplayDigits('06')
+  })
+
+  it('stops when reaches zero', function () {
+    const component = renderAnimatedTenSecondCounter()
+    jasmine.clock().tick(10000)
+    expect(component).toDisplayDigits('00')
+    jasmine.clock().tick(10000)
+    expect(component).toDisplayDigits('00')
+  })
+
+  it('changes with non-zero easing time', function (done) {
+    if (process.env.WATCH) {
+      pending(
+        `This spec in too inconsistent for the watch mode and thus is disabled.
```

Comment:
It's not 100% consistent outside watch mode, either :(
To be honest, I'm tempted to remove this spec completely, but it feels wrong not to test with non-zero transition time.